NEWS.md

```diff
@@ -109,16 +106,31 @@ Note: We're only listing outstanding class updates.
       Refinement#refined_class is deprecated and will be removed in Ruby
       3.4.  [[Feature #19714]]
 
-* SizedQueue
-
-    * SizedQueue#freeze now raises TypeError. [[Bug #17146]]
-
 * String
 
     * String#unpack now raises ArgumentError for unknown directives. [[Bug #19150]]
     * String#bytesplice now accepts new arguments index/length or range of the
       source string to be copied.  [[Feature #19314]]
 
+* Thread::Queue
+
+    * Thread::Queue#freeze now raises TypeError. [[Bug #17146]]
+
+* Thread::SizedQueue
+
+    * Thread::SizedQueue#freeze now raises TypeError. [[Bug #17146]]
+
+* Time
+
+    * Time.new with a string argument became stricter. [[Bug #19293]]
+
+        ```ruby
+        Time.new('2023-12-20')
+        #  no time information (ArgumentError)
+        Time.new('2023-12-20')
```

Comment:
This uses the same code twice, which doesn't appear intentional.  I would remove the duplicate lines.

Reply:
Ugh. Meant to show "year-month" and "year-month-day", it is a typo. But anyway, I think one example is enough.